portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,261 @@
-#content {
+:root {
+  --pink: rgb(255, 102, 196);
+  --pinkPastel: rgb(255, 173, 224);
+  --magenta: rgb(203, 108, 230);
+  --magentaPastel: rgb(225, 168, 240);
+  --purple: rgb(140, 82, 255);
+  --purplePastel: rgb(187, 153, 255);
+  --yellow: rgb(243, 210, 80);
+  --offwhite: rgb(236, 236, 236);
+  --black: rgb(28, 33, 32);
+}
+
+@font-face {
+  font-family: Spartan;
+  src: url(./fonts/LeagueSpartan-Bold.otf);
+}
+
+@font-face {
+  font-family: HK Grotesk Light;
+  src: url(./fonts/HKGrotesk-Light.otf);
+}
+
+@font-face {
+  font-family: HK Grotesk Med;
+  src: url(./fonts/HKGrotesk-Medium.otf);
+}
+
+body {
+  background-color: white;
+}
+
+h1 {
+  color: var(--black);
+  font-family: Spartan;
+  font-size: 3rem;
+}
+
+h2 {
+  font-family: HK Grotesk Light;
+  font-size: 2rem;
+  margin-bottom: 0rem;
+}
+
+h3 {
+  font-family: HK Grotesk Med;
+  font-size: 2rem;
+  margin-bottom: 0rem;
+}
+
+p {
+  font-family: HK Grotesk Med;
+  font-size: 1rem;
+}
+
+.navbar {
+  height: 15%;
+  display: flex;
+  align-items: center;
```

Comment:
Like we talked about in the meeting, the Google style guide recommends alphabetizing CSS: https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

I actually don't like this rule myself, so it's up to you how strictly you want to follow it!

Reply:
I was trying to organize these by "sections" but it's getting a little messy so I'll probably alphabetize it!